docs/tool/js/views/map-view.js

```diff
@@ -1019,12 +1019,26 @@
         addExportButton: function() {
           // Get the modal
           var modal = d3.select('#exportDataModal');
+          var filtersText = d3.select('#exportFilters');
           // Get the <span> element that closes the modal
           var span = d3.select(".close")[0];
           d3.select('#csvExportButton')
             .on('click', function(d) {
               modal.style.display = "block";
               modal.class = "modal-open";
+              var activeFilters = filterUtil.getActiveFilterValues();
+              var activeFiltersString = "";
+              Object.keys(activeFilters).forEach(function(key){
+                activeFiltersString += key + ": ";
+                for (var i = 0; i < activeFilters[key].length; i++ ){
+                  activeFiltersString += activeFilters[key][i];
+                  if ( i < activeFilters[key].length - 1 ){
+                    activeFiltersString += ",";
+                  }
+                }
+                activeFiltersString += "\n";
+              });
+              filtersText._groups[0][0].innerText = activeFiltersString;
```

Comment:
It won't be obvious to users what this means: `proj_units_tot: 4,410,true`. Instead, we should use the display_name of the filter (you can get this from the config data stored in `filterView.components` or by iterating over the ui controls themselves via `filterView.filterControls` and looking in the 'component' attribute). 

Also, when it is an array of [min,max,include_nulls] (i.e. date or continuous), we should label them as such. 

I suggest grouping the filters into groups based on `component_type` so that you can make a properly formatted table for each type - 

continuous, date: 3 columns for min, max, include_nulls
location, searchbar, categorical: list of matching categories. 

To prevent future 'missing filters' bugs in case we add new types of categories, use an 'else' at the end to catch anything else and dump it into a string as -is.